package.json

```diff
@@ -71,29 +74,45 @@
     "@electron-forge/maker-squirrel": "6.0.0-beta.53",
     "@electron-forge/maker-zip": "6.0.0-beta.53",
     "@electron-forge/plugin-webpack": "6.0.0-beta.53",
-    "@marshallofsound/webpack-asset-relocator-loader": "^0.5.0",
+    "@marshallofsound/webpack-asset-relocator-loader": "0.5.0",
     "@types/react": "16.9.49",
     "@types/react-dom": "16.9.8",
-    "@typescript-eslint/eslint-plugin": "^3.9.0",
-    "@typescript-eslint/parser": "^3.9.0",
-    "css-loader": "^4.2.1",
+    "@types/ws": "7.2.6",
+    "@typescript-eslint/eslint-plugin": "^3.6.1",
+    "@typescript-eslint/parser": "4.0.1",
+    "css-loader": "4.2.1",
     "electron": "10.1.1",
-    "eslint": "^7.6.0",
-    "eslint-plugin-import": "^2.20.0",
-    "fork-ts-checker-webpack-plugin": "^5.0.14",
-    "node-loader": "^1.0.1",
-    "style-loader": "^1.2.1",
-    "ts-loader": "^8.0.2",
+    "eslint": "7.6.0",
+    "eslint-config-airbnb-typescript": "9.0.0",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-import": "^2.22.0",
```

Comment:
Can we make all of these a specific version?

Reply:
Done